back/engines/commercial/public_api/lib/tasks/public_api_tasks.rake

```
@@ -1,5 +1,20 @@
 # frozen_string_literal: true
-# desc "Explaining what the task does"
-# task :public_api do
-#   # Task goes here
-# end
+
+desc 'Public API'
+namespace :public_api do
+  # https://www.notion.so/citizenlab/Receive-public-API-credentials-72b6a21ca8e6445f856f26a18757d1c8
+  task :create_client, [:tenant_host] => :environment do |_t, args|
+    tenant = Tenant.find_by(host: args[:tenant_host])
+
+    client = PublicApi::ApiClient.create(tenant: tenant)
```

Lets fail hard with a `create!` instead?